### PR TITLE
Add some apps

### PR DESCRIPTION
**Pull request by chettoy**
Added ADM, ADM pro, 学习通, QQ, SD Maid, VooV, 文件管理器+

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,8 @@ appops set [package name] ACTIVATE_VPN allow
 |----------------|-----------------------|---------------|---------------|-----|
 | 23andMe | 5.114.0 | 🟩 |||
 | A+ Gallery | 2.2.55.4 | 🟩 | You might face graphical glitches when using dark theme, hence its recommended to use light theme instead. ||
+| ADM (com.dv.adm) | 12.5.4 | 🟩 |||
+| ADM pro (com.dv.adm.pay) | 6.4.0 | 🟩 |||
```
Comment (pratyakshm) [inline README.md:50]:
```suggestion
| ADM | 12.5.4 | 🟩 |||
| ADM Pro | 6.4.0 | 🟩 |||
```

File: README.md [hunk 2]
```diff
@@ -74,6 +76,7 @@ appops set [package name] ACTIVATE_VPN allow
 | Candy Crush Saga | 1.213.2.1 (12132011) | 🟩 |||
 | Canvas Student | 6.14.1 | 🟩 |||
 | CarX Highway Racing | 1.17.1 | 🟨 | Terrible in-game experience, includes poor performance and low FPS ||
+| 超星学习通 (com.chaoxing.mobile) | 5.0.3 | 🟩 | Latest version crashes | Version 4.6.1 is fine |
```
Comment (pratyakshm) [inline README.md:79]:
```suggestion
| 超星学习通 | 5.0.3 |🟨| Version (5.0.3) (latest) crashes on startup, but version 4.6.1 works ||
```

File: README.md [hunk 3]
```diff
@@ -175,12 +178,14 @@ appops set [package name] ACTIVATE_VPN allow
...
 | Q-Dance | 8.0.7 | 🟥 | App crashes ||
 | QooApp | 8.3.3 | 🟩 |||
+| QQ | 8.2.11 | 🟩 |||
 | Rayman Classic | 1.0.1 | 🟩 |||
 | Reddit | | 🟩 |||
 | Relay | 10.0.378 | 🟩 |||
 | Remote Desktop (Microsoft) | 10.0.12.1148 | 🟩|||
 | Roblox | 2.499.381 | 🟨 | Graphical anomalies | GMS warnings might appear but these can be safely ignored |
 | Rootless Launcher | 3.9.1 | 🟥 | App crashes |
+| SD Maid (pro) | 5.2.2 | 🟨 | Unable to grant external storage privileges, can be skipped ||
```
Comment (pratyakshm) [inline README.md:188]: What does this exactly mean? If skipped, then the app won't be able to offer functionality regarding external storage, right? 
If so, please mention clearly.

File: README.md [hunk 4]
```diff
@@ -218,12 +223,14 @@ appops set [package name] ACTIVATE_VPN allow
 | Via Browser | 4.3.1 | 🟩 ||
 | VLC | 3.4.0 | 🟩 ||
 | VK | 6.58 | 🟩 ||
+| VooV (腾讯会议国际版, com.tencent.voov) | 2.12.5.504 | 🟩 ||
```
Comment (pratyakshm) [inline README.md:226]:
```suggestion
| VooV (腾讯会议国际版) | 2.12.5.504 | 🟩 ||
```

File: README.md [hunk 5]
```diff
@@ -218,12 +223,14 @@ appops set [package name] ACTIVATE_VPN allow
...
 | Warden | 1.0.3.release | 🟨 | App screen flashes otherwise functionality-wise its normal |
 | WhatsApp | 2.21.20.20 | 🟨 | WhatsApp cloud chat backups will not work, app was tested with microG installed |
 | Word | 16.0.14430.20246 | 🟩 || Might require microG |
 | 微博 (Weibo) | 11.10.1 | 🟨 | Cannot sign in using password, shows limit reached for verification codes |
-| 微博国际版 （Weibo International) | 3.9.8 | 🟨 | Cannot sign in |
+| 微博国际版 (Weibo International) | 3.9.8 | 🟨 | Cannot sign in |
 | 微博极速版 (Weibo Fast) | 10.9.2 (4620) | 🟨 | Cannot sign in |
+| 文件管理器+ (com.alphainventor.filemanager) | 2.7.1 | 🟩 ||
```
Comment (pratyakshm) [inline README.md:233]:
```suggestion
| 文件管理器+ | 2.7.1 | 🟩 ||
```

**Review by pratyakshm — CHANGES_REQUESTED** [review]
Hi - thank you for contributing. 
We currently don't include the package names so I have removed them for now. Please let us know if this looks good.

File: README.md (hunk 2 above)
Comment (riverar) [reply README.md:79]: If it crashes on startup, then this should be red and the note simply "Crashes on startup". A separate entry should be created for 4.6.1.

Comment (pratyakshm) [reply README.md:79]: > If it crashes on startup, then this should be red and the note simply "Crashes on startup". A separate entry should be created for 4.6.1.

looks good now?

Comment (riverar) [reply README.md:79]: Perfect.

**Pull request merged**